locales/es.yml

```diff
@@ -192,6 +191,7 @@ es:
     email_required: "Se requiere un correo"
     email_username_in_use: "Correo o nombre en uso."
     INVALID_ASSET_URL: "La URL del articulo no es valida"
+    unexpected_error: "Lo siento. Ha habido un error no previsto."
```

Comment:
I noticed in `locales/en.yml`, `unexpected_error` was changed to `unexpected`.
https://github.com/coralproject/talk/blob/9de99a9ef45fe89e345af6951beb2f456fb7f39b/locales/en.yml#L209

I also noticed `msg = t('error.unexpected')` in `forEachError`.
https://github.com/coralproject/talk/blob/9de99a9ef45fe89e345af6951beb2f456fb7f39b/client/coral-framework/utils/index.js#L142.

Did you mean to change the key here to `unexpected` as well?